diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6a9a694..89a33a6 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -70,7 +70,8 @@ target_compile_definitions(${PROJECT_NAME}
 
 target_include_directories(${PROJECT_NAME}
   PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
-         $<INSTALL_INTERFACE:include>)
+         $<INSTALL_INTERFACE:include>
+  PRIVATE $<$<PLATFORM_ID:Windows>:${CMAKE_CURRENT_SOURCE_DIR}/include/win32>)
 
 target_link_libraries(${PROJECT_NAME}
   PUBLIC $<$<NOT:$<C_COMPILER_ID:MSVC>>:m>)
